README.md

```diff
@@ -19,3 +55,11 @@ Add the following line to your gemfile
 ```ruby
 gem 'dill', github: 'mojotech/dill'
 ```
+
+
+## Contributing
+We welcome contributions! Email Adam at ags@mojotech.com if you're interested.
+
+
+## About MojoTech
+We build software that grows businesses. Learn more about us at [our website](www.mojotech.com) or follow us on [our blog](http://blog.mojotech.com/).
```

Comment:
Based on how we tag https://github.com/mojotech/jeet, might we add a note about hiring?